lib/tasks.js

```diff
@@ -58,12 +58,12 @@ exports.tasks = [];
  * // Add the task to Lando
  * lando.tasks.add('destroy', task);
  */
-exports.add = function(name, task) {
-
-  // Build a task object
-  var tasker = _.merge(task, {name: name});
-
-  // Add to the task
-  this.tasks.push(tasker);
+exports.add = (name, task) => {
+  const mergedTask = _.merge(task, {name: name});
+  // Minimal task validation, just checking that common properties exist. If not, throw error.
+  if (!(_.has(mergedTask, 'name') && _.has(mergedTask, 'command') && _.has(mergedTask, 'describe') && _.has(mergedTask, 'run'))) {
```

Comment:
@dustinleblanc not a block to merge but there's gotta be a ... shorter way to do this? 

```js
const requiredKeys = ['name', ... 'whatever'];
if (!_.isEmpty(_.difference(_.intersection(requiredKeys, _.keys(mergedTask), requiredKeys))
```
but better

Reply:
Yeah I almost threw it into its own function, also wanted to method chain like
```js
_(mergedTask).has('foo').has('bar)
```
I'd actually prefer:

```js
if(!validTask(task))
```
And just do the dirty elsewhere...

Reply:
Which actually, ruby has a convention of suffixing boolean functions with `?`

I'd absolutely love:
```js
unless validTask?(task) 
  throw a Fit()
```